docs/apb-filter-design.md

```diff
@@ -0,0 +1,80 @@
+# APB Filter Design Doc
+
+## High Level Summary
+
+Introduce 2 new fields to broker section of config:
+
+`whitelist: "/etc/ansible-service-broker/whitelist.yaml"`
+`blacklist: "/etc/ansible-service-broker/blacklist.yaml"`
+
+Both are optional file paths that are yaml files containing arrays
+of regexes. Broker will construct the set of eligible APBs from this list.
+
+If only whitelist is present, strictest mode of operation. Any APB that does *not*
+match on a regex in the whitelist will be filtered.
+
+If only blacklist is present, all discovered APBs will be eligible *unless*
+they match on a regex found in the blacklist.
+
+If both are present, same behavior as whitelist, but if a value matches on white
+and black, blacklist will take precedence and block the whitelist match.
+
+These can be easily configured by cluster operators via config maps.
```

Comment:
This is not intended to be considered operator documentation, I need to follow this up with that. It was mostly scratch design thoughts prior to writing any code.